Cargo.toml

```diff
@@ -1,28 +1,18 @@
 [workspace]
-members = []
+members = ["tools"]
```

Comment:
Don't you need to make `rav1d` the library a member, too?  That was my understanding of how workspaces work at least.

Reply:
Yeah I think this needs to be

```toml
[workspace]
members = [".", "tools"]
default-members = [".", "tools"]
```

Otherwise the root rav1d library doesn't get built (or only gets built as a dependency of the CLI tools)

Reply:
Adding this may also mean you don't need to add the feature flags I suggested in [this comment](https://github.com/memorysafety/rav1d/pull/1308#issuecomment-2223740522), I'm not 100% sure how dependency features interact with workspaces in this case.

Reply:
I think it'd be better to do some restructuring of the files at this point.  Move `src/`, `include`, and `lib.rs` under `rav1d`.  Having the top level be a package that contains other packages which depend on it is IMO very confusing.

Reply:
Not in this PR, and we have other higher priority things to worry about first. We need to do a general re-org and separate and document the C API, but not this week.